src/_assets/css/_base.scss

```diff
@@ -41,3 +38,26 @@ dd {
 .material-icons {
   font: $site-font-icon;
 }
+
+.site-image-right {
+  display: block;
+  margin: 0 auto;
+  max-height: 60vh;
+  max-width: 100%;
+
+  @include media-breakpoint-up(md) {
+    float: right;
+    margin: 0 0 bs-spacer(10) bs-spacer(10);
+    max-height: none;
+    max-width: calc((100%/2.5) - #{bs-spacer(10)});
+  }
+}
+
+.site-figure {
+  display: flex;
+  justify-content: center;
+
+  figcaption{
+    margin-top: bs-spacer(4);
```

Comment:
Center the caption text? (If so, remove the `text-center` occurrences in the page sources.)

Reply:
I added the centered style to the figcaption now.